x/validator-preference/keeper.go

```diff
@@ -0,0 +1,77 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/tendermint/tendermint/libs/log"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/gogo/protobuf/proto"
+	"github.com/osmosis-labs/osmosis/v12/osmoutils"
+	"github.com/osmosis-labs/osmosis/v12/x/validator-preference/types"
+)
+
+type Keeper struct {
+	storeKey            sdk.StoreKey
+	paramSpace          paramtypes.Subspace
+	stakingKeeper       types.StakingInterface
+	bankKeeper          types.BankInterface
+	communityPoolKeeper types.CommunityPoolInterface
+}
+
+func NewKeeper(storeKey sdk.StoreKey,
+	paramSpace paramtypes.Subspace,
+	stakingKeeper types.StakingInterface,
+	bankKeeper types.BankInterface,
+	communityPoolKeeper types.CommunityPoolInterface,
```

Comment:
same question here, asking since I've noticed that we're passing in the Distr Keeper from  app/keepers.go